### ISSUE TITLE
Radios and checkboxes do not follow native colours in Windows High Contrast

### ISSUE THREAD

**Issue by adamliptrot-oc** [alphagov/govuk-frontend#2258]
## Description of the issue
There seems to be a discrepancy between how native radios are displayed in Windows High Contrast and how GOVUK ones are - the highlight colour used on native inputs (including the text boxes in the Design System) is missing from the focus and selected states of radios (and checkboxes), probably down to the way in which the pseudo control is built. This makes the focus state somewhat less visible than the native version.

Screenshot shows Design System with focussed (top), selected and unselected/unfocussed vs native elements on the right (focussed at top, selected below).
<img width="703" alt="Screenshot 2021-06-24 at 08 06 01" src="https://user-images.githubusercontent.com/14074579/123237174-51c63980-d4d5-11eb-8856-6e4c7095d4b9.png">

Screenshot to show Design System text input focus state
<img width="734" alt="Screenshot 2021-06-24 at 10 15 48" src="https://user-images.githubusercontent.com/14074579/123237232-630f4600-d4d5-11eb-919a-86f01d3759af.png">

## Steps to reproduce the issue
Open https://design-system.service.gov.uk/components/radios/ in MS Edge on Windows and enable Windows High Contrast Mode

## Actual vs expected behaviour
Expectation is that the Design System radios and checkboxes to follow the native colouring in line with the Design System's text inputs, but they use white instead of the blue highlight which makes them much less obvious as the default state is also white.

## Environment (where applicable)
- Operating system: Windows 10
- Browser: MS Edge
- Browser version: 91.0.864.54
- GOV.UK Frontend Version: 3.12.0

**Comment by 36degrees** [comment]
Had a quick play with this and it looks like we might be able to use the `Highlight` system colour keyword to add an outline that's consistent with the focus outline used for native inputs.

For example:

```scss
  // Focused state
  .govuk-checkboxes__input:focus + .govuk-checkboxes__label:before {
    border-width: 4px;
    box-shadow: 0 0 0 $govuk-focus-width $govuk-focus-colour;

    @media screen and (forced-colors: active) {
      outline: $govuk-focus-width solid highlight;
      outline-offset: 1px;
    }
  }
```

![Screenshot 2021-06-24 at 11 50 13](https://user-images.githubusercontent.com/121939/123250759-57764c00-d4e2-11eb-8985-3a386bdef8d4.png)

The focus state on radios unfortunately is not as neat as there's no `outline-radius`.

![Screenshot 2021-06-24 at 11 54 46](https://user-images.githubusercontent.com/121939/123251323-fac76100-d4e2-11eb-9f79-cf2304967ee3.png)

It _might_ be possible to use `forced-color-adjust: none` and use the box shadow to take more control over the radio focus style, but we'd need to be very careful to respect the forced color styles and test thoroughly…

If we wanted to support older browsers that don't support the `forced-colors` media query, we could also add a transparent outline outside of the media query which would work in these older browsers, but it'd match currentColor (in this example, a white outline) rather than matching the other focus styles. I believe we used to do this before the changes in #1316.
